Gemfile

```
@@ -102,3 +105,4 @@ group :test do
   gem "simplecov", require: false
   gem "spring-commands-rspec"
 end
+gem "graphiql-rails", group: :development
```

Can you move to above☝🏻?
```suggestion
gem "graphiql-rails"
```